index.bs

```diff
@@ -1224,52 +1224,80 @@ as the *sender* and the agent receiving the media as the *receiver*.
 
 Capabilities {#streaming-capabilities}
 --------------------------------------------
+If the advertiser is already authenticated, the requester has the ability to
+request additional information by sending an streaming-capabilities-request
+message, and receive back a streaming-capabilities-response message with the
+following properties:
 
-<!--
-  TODO:
+: receive-audio (required)
+:: A list of audio capabilities. For an explanation of fields, see below.
 
-  Put in something like this:
+: receive-video (required)
+:: A list of video capabilities. For an explanation of fields, see below.
 
-streaming-capabilities-request {
-  request
-}
+: receive-data (required)
+:: A list of arbitrary data formats (represented as codecs) the device supports
+    for receiving data.
 
-streaming-capabilities-response {
-  response
-  capabilities: streaming-capabilities
-}
+The codec type is used as the basis for audio, video, and data capabilities.
+Codes are composed of the following properties:
 
-streaming-capabilities = {
-  receive-audio: [* receive-audio-capability]
-  receive-video: [* receive-video-capability]
-  receive-data: [* receive-data-capability]
-}
+: codec-name (required)
+:: The name of the codec.
 
+: parameters (required)
+:: A list of (key, value) parameters that can be used to transmit arbitrary
+    information about a codec.
```

Comment:
Should say something about these being for codec-specific parameters that aren't applicable to all codecs of a certain media type (audio, video, data).

Reply:
agreed.